packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -357,6 +369,38 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
                 });
             }
         });
+        registry.registerCommand(OpenEditorsCommands.CLOSE_ALL_TABS_FROM_TOOLBAR, {
+            execute: widget => this.withOpenEditorsWidget(widget, () => this.editorWidgets.forEach(editor => editor.close())),
+            isEnabled: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length),
+            isVisible: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length)
+        });
+        registry.registerCommand(OpenEditorsCommands.SAVE_ALL_TABS_FROM_TOOLBAR, {
+            execute: widget => this.withOpenEditorsWidget(widget, () => this.commandService.executeCommand(CommonCommands.SAVE_ALL.id)),
```

Comment:
No need for the `CommandService` injection:

```suggestion
            execute: widget => this.withOpenEditorsWidget(widget, () => registry.executeCommand(CommonCommands.SAVE_ALL.id)),
```

Reply:
Good to know, I've fixed this 👍